### PR TITLE
[WIP] Stub version of README.md, adapting to Sphinx-based workflow

### PR DESCRIPTION
**Pull request by ajpowelsnl**
This is a test.  This commit will be tailored to our new workflow.

### CODE REVIEW

File: source/README_SPHINX.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Kokkos Documentation Workflow <br/><br/>
+
+The kokkos-core-wiki is the canonical source of Kokkos documentation "Truth".  Please do not directly edit the main Kokkos project Wiki.<br/>
+
+Sandia employees: Creation of wholly new pages will require Review and Approval (R&A).  Check with your Sandia team lead if you have questions about the R&A process.<br/><br/>
+
+* Step 1) Fork the kokkos-core-wiki repo (`https://github.com/kokkos/kokkos-core-wiki`) if you do not yet have your own fork.<br/>
+
+* Step 2) Clone your fork and add upstream `kokkos-core-wiki` repo as a remote (`git clone git@github.com:ajpowelsnl/kokkos-core-wiki.git`, `git remote add upstream git@github.com:kokkos/kokkos-core-wiki.git`) if you have not already done so. If you already have a fork of the `kokkos-core-wiki`, then simply update your fork and checkout the branch we'll use for transitioning the documentation (`git fetch upstream -p`, `git checkout upstream/sphinx_porting_stage1`).<br/>
```
Comment (fnrizzi) [inline source/README_SPHINX.md:9]: I would highlight the branch name more since, at least for the time being, all new PRs should be posted against that

**Comment by fnrizzi** [comment]
@ajpowelsnl steps 8,9,10,11,12 are outdated. 

We basically just need:
- make fork if needed and set upstream remote
- checkout proper branch `git checkout upstream/sphinx_porting_stage1`
- make a new branch with some name 
- edit files inside `source/` subdirectory 
- build: `make html` and view the `html/index.html` in a browser 
- push changes and make PR

**Comment by fnrizzi** [comment]
@ajpowelsnl if you are revising this, please make sure you check the latest changes to the branch

**Comment by fnrizzi** [comment]
a lot has changed in the sphinx_porting_stage1 branch so please rebase

**Comment by fnrizzi** [comment]
@ajpowelsnl are you still on this or should we just close without mergin?

**Comment by ajpowelsnl** [comment]
> @ajpowelsnl are you still on this or should we just close without mergin?


@fnrizzi -- Apologies for my delayed response.  I was away for a week, and am re-focusing on this topic.  I would like to merge the changes.  I'll review all comments, and get back to you once I figure out the best way forward.

**Comment by ajpowelsnl** [comment]
Keep It Simple, Simon

**Pull request closed without merging**